api-inference-community/docker_images/keras/app/pipelines/image_classification.py

```diff
@@ -45,8 +66,9 @@ def __call__(self, inputs: "Image.Image") -> List[Dict[str, Any]]:
         expected_input_size = self.model.input_shape
         if expected_input_size[-1] == 1:  # Single channel, we assume grayscale
             inputs = inputs.convert("L")
+
         target_size = (expected_input_size[1], expected_input_size[2])
-        img = inputs.resize(target_size)
+        img = inputs.resize(target_size, self.interpolation)
```

Comment:
Not sure if interpolation is needed, but its in [Keras's `load_img` function](https://github.com/keras-team/keras-preprocessing/blob/6701f27afa62712b34a17d4b0ff879156b0c7937/keras_preprocessing/image/utils.py#L76-L172), so I decided to add it.